### ISSUE TITLE
Model Taking an exceedingly long time to compile/run

### ISSUE THREAD

**Issue by mhamilton723** [farizrahman4u/seq2seq#15]
Hey, I'm trying to run your model on tensors of the shape 
X=(16041, 100, 2)  and Y=(16041, 50, 2)
The model is taking upwards of hours to compile (I have yet to get it to successfully compile) on FAST_RUN

on FAST_COMPILE the model takes 3 million seconds + for one epoch. Is this normal behavior? Do you have an example of the code working on a real dataset in a timely fashion? Thanks!

**Comment by farizrahman4u** [comment]
What was your batchsize?

**Comment by mhamilton723** [comment]
the variable ops.batch_size = 20; what is the difference between your models batch size and the one used in keras' fitting?

maxlen = 100 #length of input sequence and output sequence
                hidden_dim = 500 #memory size of seq2seq
                seq2seq = Seq2seq(input_length=X_train.shape[1], input_dim=X_train.shape[2], hidden_dim=hidden_dim,
                                    output_dim=X_train.shape[2], output_length=y_train.shape[1], batch_size=ops.batch_size, depth=4)

                model = Sequential()
                model.add(seq2seq)
                model.compile(loss="mean_squared_error", optimizer="rmsprop")
            else:
                raise ValueError('Not a legal model name')

            model.compile(loss="mean_squared_error", optimizer="rmsprop")
            print('Training model...')
            early_stopping = EarlyStopping(monitor='val_loss', patience=ops.patience, verbose=0)
            model.fit(X_train, y_train, batch_size=ops.batch_size, nb_epoch=ops.max_epochs,
                      validation_split=0.1, callbacks=[early_stopping])

**Comment by farizrahman4u** [comment]
Your model is big enough. I will see what I can do.
